### PR TITLE
chore(ONYX-1434): change Curators’ Picks title

### PR DESCRIPTION
**Pull request by dariakoko**
The type of this PR is: **chore**

This PR solves [ONYX-1434]

### Description

[As requested](https://artsy.slack.com/archives/C07PRTJSD6G/p1733334824005519), changing the title from "Curators’ Picks: Emerging" to "Curators’ Picks"

[ONYX-1434]: https://artsyproduct.atlassian.net/browse/ONYX-1434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/Home/Components/HomeEmergingPicksArtworksRail.tsx [hunk 1]
```diff
@@ -37,10 +37,10 @@ export const HomeEmergingPicksArtworksRail: React.FC<React.PropsWithChildren<Hom
   return (
     <ArtworkGridContextProvider hideSignals>
       <Rail
-        title="Curators’ Picks: Emerging"
+        title="Curators’ Picks"
         subTitle="The best works by rising talents on Artsy, all available now."
         viewAllLabel="View All Works"
-        viewAllHref="/collection/curators-picks-emerging"
+        viewAllHref="/collection/curators-picks-emerging" // here
```
Comment (olerichter00) [inline src/Apps/Home/Components/HomeEmergingPicksArtworksRail.tsx:43]: 👀 

```suggestion
        viewAllHref="/collection/curators-picks-emerging"
```

File: src/Apps/Home/Components/HomeEmergingPicksArtworksRail.tsx [hunk 2]
```diff
@@ -23,9 +23,9 @@ interface HomeEmergingPicksArtworksRailProps {
   viewer: HomeEmergingPicksArtworksRail_viewer$data
 }
 
-export const HomeEmergingPicksArtworksRail: React.FC<React.PropsWithChildren<HomeEmergingPicksArtworksRailProps>> = ({
-  viewer,
-}) => {
+export const HomeEmergingPicksArtworksRail: React.FC<React.PropsWithChildren<
+  HomeEmergingPicksArtworksRailProps
+>> = ({ viewer }) => {
```
Comment (olerichter00) [inline src/Apps/Home/Components/HomeEmergingPicksArtworksRail.tsx:28]: `PropsWithChildren` should not be needed.

```suggestion
export const HomeEmergingPicksArtworksRail: React.FC<HomeEmergingPicksArtworksRailProps> = ({ viewer }) => {
```

File: src/Apps/Home/Components/HomeEmergingPicksArtworksRail.tsx [hunk 3]
```diff
@@ -121,7 +121,9 @@ export const HomeEmergingPicksArtworksRailFragmentContainer = createFragmentCont
   }
 )
 
-export const HomeEmergingPicksArtworksRailQueryRenderer: React.FC<React.PropsWithChildren<unknown>> = () => {
+export const HomeEmergingPicksArtworksRailQueryRenderer: React.FC<React.PropsWithChildren<
+  unknown
+>> = () => {
```
Comment (olerichter00) [inline src/Apps/Home/Components/HomeEmergingPicksArtworksRail.tsx:126]: The same here

File: src/Apps/Marketing/Routes/MarketingFindArtYouLoveRoute.tsx [hunk 4]
```diff
@@ -18,7 +18,9 @@ import { MarketingCollectionCell } from "Apps/Marketing/Components/MarketingColl
 import { MarketingQuizCTA } from "Apps/Marketing/Components/MarketingQuizCTA"
 import { BRAND_PALETTE } from "Apps/Marketing/Utils/brandPalette"
 
-export const MarketingFindArtYouLoveRoute: FC<React.PropsWithChildren<unknown>> = () => {
+export const MarketingFindArtYouLoveRoute: FC<React.PropsWithChildren<
+  unknown
+>> = () => {
```
Comment (olerichter00) [inline src/Apps/Marketing/Routes/MarketingFindArtYouLoveRoute.tsx:23]: here as well.

**Review by olerichter00 — DISMISSED** [review]
🌟

File: src/Apps/Home/Components/HomeEmergingPicksArtworksRail.tsx (hunk 2 above)
Comment (dariakoko) [reply src/Apps/Home/Components/HomeEmergingPicksArtworksRail.tsx:28]: Hm, not sure why it's there, i didn't make this change

Comment (dariakoko) [reply src/Apps/Home/Components/HomeEmergingPicksArtworksRail.tsx:28]: I see we use PropsWithChildren in a lot of places. Will create a ticket to address it separately

**Pull request merged**